package.json

```diff
@@ -19,7 +19,9 @@
     "test": "jest ."
   },
   "peerDependencies": {
-    "eslint": "^5.15.1",
+    "@babel/core": "^7.4.0",
+    "babel-eslint": "^10.0.0",
+    "eslint": "^5.3.0",
```

Comment:
[eslint-config-airbnb-base](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/package.json#L67) is constrained to this version.

Reply:
while airbnb-base has ^5.3.0, I'm pretty sure we can still have this library reference the latest minor version, since they've elected to use `^`. did you see any issues specifically?

Reply:
so I run the following command to install the peer dependecies
```bash
npx install-peerdeps --dev eslint-config-ebates
```
and `eslint` 5.3.0 is installed. Then `yarn` complains because we as for `5.15.1`.